rclcpp/include/rclcpp/generic_publisher.hpp

```diff
@@ -109,9 +110,11 @@ class GenericPublisher : public rclcpp::PublisherBase
     }
   }
 
+  RCLCPP_PUBLIC
```

Comment:
do you need them even when defaulted?

Reply:
I'm not sure, but I see them in other places with defaulted constructors.